kytos/utils/config.py

```diff
@@ -10,14 +10,41 @@
 import logging
 import os
 import re
+import shutil
 from collections import namedtuple
 from configparser import ConfigParser
+from pathlib import Path
 from urllib.error import URLError
 from urllib.request import urlopen
 
+BASE_ENV = os.environ.get('VIRTUAL_ENV', '/')
+ETC_KYTOS = 'etc/kytos'
+KYTOS_SKEL_PATH = 'templates/skel'
```

Comment:
If those variables will be used only by the `create_skel_dir()` function they don't need to be globals.